ui/components/component-library/button-link/README.mdx

```diff
@@ -13,7 +13,7 @@ The `ButtonLink` is an extension of `ButtonBase` to support link styles
 
 ## Props
 
-The `ButtonLink` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) and [ButtonBase](/docs/components-componentlibrary-buttonbase--default-story#props) component props
+The `ButtonLink` accepts all props below as well as all [Box](/docs/components-componentlibrary-box--docs#props) and [ButtonBase](/docs/components-componentlibrary-buttonbase--default-story#props) component props
```

Comment:
Let's remove this line altogether now that the props table shows props for the `Box` and `BaseButton`. We can also remove lines `20 - 22`

```suggestion
```

```
The `ButtonLink` accepts all [ButtonBase](/docs/components-componentlibrary-buttonbase--default-story#props) component props
<ArgsTable of={ButtonBase} />
```